WordPress/gutenberg-starter-theme
# Migrate Gutenberg theme support options into checkboxes.

This PR takes the optional Gutenberg theme support declarations that the Gutenberg Starter theme uses, and migrates them into checkbox options on the "Theme Options" settings page created in #85. (Note that this page now lives under the "Appearance" menu). Moving these here will provide a quick way for developers to enable and disable these options while testing Gutenberg. 

- `align-wide`
- `editor-color-palette`
- `wp-block-styles` (the "opinionated" styles required to support the core blocks in the front end.)
- `responsive-embeds`

Because all of these had previously been enabled by default, the new checkboxes default to checked. 

## Testing

- Install the PR version of this theme on a test site, do not change any settings yet.
- Ensure that the options above are all active by default:
	- You should see wide and full alignment options available in the editor. 
	- You should see the custom theme color palette available in the editor.
	- You should see default block styles applied on the frontend. (A quick way to check is to verify that the quote block appears with a left border).  
	- On the front end, the body should have a `wp-embed-responsive` class. 
- Check and uncheck the new and verify that they toggle the options as expected. 

---

**Screenshot:** 

<img width="744" alt="Screen Shot 2019-07-24 at 9 02 02 AM" src="https://user-images.githubusercontent.com/1202812/61795584-c11ae680-adf1-11e9-8428-7d509edd5610.png">
